src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyActions.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+
+type Props = {
+  onCancel: () => void;
+  onSave: () => void;
+};
+
+const ProjectDataPrivacyActions = ({onCancel, onSave}: Props) => (
+  <Wrapper>
+    <StyledButton onClick={onCancel}>{t('Cancel')}</StyledButton>
+    <StyledButton priority="primary" onClick={onSave}>
+      {t('Save Rules')}
+    </StyledButton>
+  </Wrapper>
+);
+
+export default ProjectDataPrivacyActions;
+
+const Wrapper = styled('div')`
+  display: flex;
+  justify-content: flex-end;
+  > * {
+    margin-right: 14px;
+    :last-child {
```

Comment:
Should we use grid instead?

Reply:
good idea 💡

Reply:
:+1: to using grid here, as you can use grid-column-gap to add your margins.

Reply:
You might also want to use the `ButtonBar` component as it handles this kind of layout.